filechanges/opt/musicbox/setsound.sh

```diff
@@ -179,33 +179,56 @@ then
 cat << EOF > /etc/asound.conf
 pcm.!default {
     type plug
+    slave.pcm plugequal;
+}
+pcm.plugequal {
+    type dmix
+    ipc_key 1024
+    controls "/home/mopidy/.alsaequal.bin"
```

Comment:
The .alsaequal.bin file should be created automatically on the first reboot after the equalizer module is installed.

The file is used to store the user-defined equalizer profile that can be configured with ```alsamixer -D equal```